src/main/java/bio/terra/cli/serialization/userfacing/resource/UFGcsBucket.java

```diff
@@ -20,21 +21,30 @@
 public class UFGcsBucket extends UFResource {
   public final String bucketName;
   public final String location;
+  public final Integer numObjects;
+
+  // the maximum number of objects to iterate through in the bucket.
+  // if there are more, we just add a "+" at the end for display
```

Comment:
Optional, probably overthinking it, but did you consider using the `LINES` envvar if set, otherwise default to `100`?

Reply:
I thought the `LINES` is typically used for terminal/window size. That doesn't seem to apply here, since we're not displaying the bucket objects, just the count. Have you seen it used in other circumstances?

Reply:
Ah yes, I misinterpreted that we were going to print object ID's.  Disregard.